src/EditorFeatures/CSharpTest/Structure/StringLiteralExpressionStructureTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Structure;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Structure;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Structure
+{
+    public class StringLiteralExpressionStructureTests : AbstractCSharpSyntaxNodeStructureTests<LiteralExpressionSyntax>
+    {
+        internal override AbstractSyntaxStructureProvider CreateProvider() => new StringLiteralExpressoinStructureProvider();
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Outlining)]
+        public async Task TestVerbatimString()
+        {
+            await VerifyBlockSpansAsync(
```

Comment:
Please also add tests for single line string literal and erroneous case like `string x = @";`

Reply:
>single line string literal 

Looks like that is covered elsewhere for all structure providers, note we do report outlining for single line blocks, e.g. the following passes.
```cs
class C
{
    {|hint:$$namespace N{|textspan:{}|}|}
}
```